p5-travel-app/src/client/js/geoNamesApi.js

```diff
@@ -0,0 +1,27 @@
+const username = "leemoispace";
+
+export async function getGeoCity(city) {
+  const dataUrl =
+    "http://api.geonames.org/" +
+    "searchJSON?formatted=true&q=" +
+    city +
+    "&username=" +
+    username +
+    "&style=full";
+  try {
+    const res = await fetch(dataUrl);
+    if (res.ok) {
+      const city = {};
+      const data = await res.json();
+      //console.log(data);
+      city.latitude = data.geonames[0].lat;
+      city.longitude = data.geonames[0].lng;
+      city.population = data.geonames[0].population;
+      city.countryCode = data.geonames[0].countryCode;
+      console.log(city);
```

Comment:
keep console in your local, don’t commit this line

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)